### DISCUSSION TITLE
Scoop use

### DISCUSSION THREAD

**Discussion by onthebusiness** [ScoopInstaller/Scoop#5523] [Help and Questions]
How i can specify used binary, like a asdf-use?

**Comment by HUMORCE** [comment]
Please describe in detail what you want to do

**Reply by onthebusiness** [reply]
I have multiple versions of clang installed from different manifests and want to use specific binary.

How can i specify? 

And yet, how to make sure that the shim does not change when updating?

**Comment by HUMORCE** [comment]
`scoop help reset` and `scoop help shim`.
